src/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './AutoBus';
```

Comment:
I think we should export some more stuff. E.g. interfaces and such. For example if we want to use autobus, but with a custom `Transport`.